### DISCUSSION TITLE
style(item) 这个写法没找到对应的方法，可以解释一下这个吗？

### DISCUSSION THREAD

**Discussion by jackvale** [Tencent/tdesign-miniprogram#2080] [Q&A] [answered]
在common > utils.wxs > function _style(styles) 里面的 108行 ：        ` return isArray(item) ? style(item) : endsWith(item, ';');`  这里面的style(item) 不是很懂，希望可以解释一下这个方法。

**Comment by LeeJim** [comment, marked as answer]
@anlyyao

**Comment by anlyyao** [comment]
@jackhawks 递归  https://github.com/Tencent/tdesign-miniprogram/blob/673dee2fa5222073273f81ee42b2ce08ef1339cb/src/common/utils.wxs#L108
